src/pybind/mgr/cephadm/module.py

```diff
@@ -445,30 +445,6 @@ class CephadmOrchestrator(orchestrator.Orchestrator, MgrModule,
             default=False,
             desc='Log all refresh metadata. Includes daemon, device, and host info collected regularly. Only has effect if logging at debug level'
         ),
-        Option(
-            'prometheus_web_user',
-            type='str',
-            default='admin',
-            desc='Prometheus web user'
-        ),
-        Option(
-            'prometheus_web_password',
```

Comment:
want to point you towards https://github.com/ceph/ceph/pull/43889. Note that it had an explicit migration to move the old settings over to theconfig-key store. Do we need something like that here? As in, if I has these set up and then upgraded to a version with this change, would my user/password be preserved or would they get set back to default?

Reply:
As discussed in the last cephadm weekly we will not be providing migrations for this brand new feature.